test/BUILD.gn

```diff
@@ -271,6 +270,12 @@ test("brave_unit_tests") {
     "//services/preferences/public/cpp",
   ]
 
+  if (enable_extensions) {
+    deps += [ "//extensions/common:common_constants" ]
+  } else if (is_android || is_ios) {
+    deps += [ "//brave/extensions:common" ]
```

Comment:
this is duplicating the logic for the brave/extensions:common target so we should just be using `//brave/extensions:common` as a dep here

Reply:
Oops! True... fixed in https://github.com/brave/brave-core/commit/e98682dac6d30c3ede265751c60dbc15300e460e